cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CcdCase.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+public class CcdCase {
```

Comment:
just case?

Reply:
Just `Case or `CCDCase`, abbreviations shouldn't be lowercased according to naming conventions.

Reply:
done